### ISSUE TITLE
Files do not appear to be committed in SVN during deployment.

### ISSUE THREAD

**Issue by herewithme** [10up/actions-wordpress#10]
Hello !

Thank you for your tool, very convenient.

I use it for my open-source plugin simple-tags:
https://wordpress.org/plugins/simple-tags/

I add your action : 
https://github.com/BeAPI/simple-tags/actions

But I encountered a problem with the tag 2.5.2
https://github.com/BeAPI/simple-tags/tree/2.5.2

My main file say : https://github.com/BeAPI/simple-tags/blob/2.5.2/simple-tags.php
Version: 2.5.2

But the automatic action has missed some files, whose main file :
https://github.com/BeAPI/simple-tags/runs/119276034

You can check the SVN log : 
https://plugins.trac.wordpress.org/log/simple-tags/

And the Github action changeset : https://plugins.trac.wordpress.org/changeset?sfp_email=&sfph_mail=&reponame=&new=2080937%40simple-tags&old=2080871%40simple-tags&sfp_email=&sfph_mail=

So, my question, is it a bug, or am I using your tool incorrectly? :)

A big thank you for your time!

**Comment by herewithme** [comment]
I say that the problem has not been repeated since.

**Comment by helen** [comment]
Hi - it would help me a lot if you could explain what exactly is missing. I don't know if it's that the output isn't what you expected because of the way `svn copy` works or if there are actual files missing for you.

**Comment by herewithme** [comment]
Hello @helen 

Strangely, the main file of the extension, simple-tags.php was not included by the script, neither during the update of the TRUNK, nor during the SVN COPY. (for the tag 2.5.2)

I have not reproduced the problem since.

**Comment by herewithme** [comment]
I manually fixed in SVN:
https://plugins.trac.wordpress.org/changeset?sfp_email=&sfph_mail=&reponame=&new=2081105%40simple-tags&old=2080937%40simple-tags&sfp_email=&sfph_mail=

**Comment by helen** [comment]
Hmmm that is definitely odd, it does seem like at the point the workflow ran your version was indeed `2.5.2` in `simple-tags.php`. I wonder if maybe `rsync` didn't detect it as a changed file somehow since the changes were so small, that's really the only thing I can think of. It may be worth adding the `-c` flag there for more robust change detection. I am glad it hasn't failed since! Let me know if you ever reproduce it.

**Comment by herewithme** [comment]
Indeed, it's strange, no bug since. I'll keep you informed ;)
I will reopen the ticket if it happens again;)

Thanks again for the project;)

**Comment by helen** [comment]
Reopening this because I managed to trigger it myself and it was what I suspected! Despite rsync allegedly also checking file mod times because version changes typically end up with the same number of characters it doesn't pick it up to be transferred. I added the `-c` flag (checksum) which is slightly slower but likely not noticeable in this situation, and more robust. Will fix this in the plugin deploy action shortly!
